src/nls/root/strings.js

```diff
@@ -800,5 +800,6 @@ define({
     "DESCRIPTION_PURE_CODING_SURFACE"                : "true to enable code only mode and hide all other UI elements in {APP_NAME}",
     "DESCRIPTION_INDENT_LINE_COMMENT"                : "true to enable indenting of line comments",
     "DESCRIPTION_RECENT_FILES_NAV"                   : "Enable/disable navigation in recent files",
-    "DESCRIPTION_LIVEDEV_WEBSOCKET_PORT"             : "Port on which WebSocket Server runs for Live Preview"
+    "DESCRIPTION_LIVEDEV_WEBSOCKET_PORT"             : "Port on which WebSocket Server runs for Live Preview",
```

Comment:
Preexisting but "DESCRIPTION_LIVEDEV_WEBSOCKET_PORT" probably should have been "DESCRIPTION_LIVE_DEV_WEBSOCKET_PORT" to be consistent with other strings.